workbench-core/user-management/src/plugins/cognitoUserManagementPlugin.ts

```diff
@@ -491,6 +523,9 @@ export class CognitoUserManagementPlugin implements UserManagementPlugin {
         }
         throw new PluginConfigurationError(error.message);
       }
+      if (error.name === 'InvalidParameterException') {
+        throw new InvalidPaginationTokenError('Invalid Pagination Token');
```

Comment:
Nit: I imagine Cognito returns this same exception if something like Limit is given an invalid value. Could we just bubble up error.message here?